gcp_variant_transforms/transforms/merge_headers.py

```diff
@@ -42,5 +190,13 @@ def extract_output(self, merged_headers):
 class MergeHeaders(beam.PTransform):
   """A PTransform to merge VCF file headers."""
 
+  def __init__(self, split_alternate_allele_info_fields=True):
```

Comment:
Please add an args section about the new split_alternate_allele_info_fields value and mention why it's needed here (e.g. changes the header compatibility rules as it changes the schema).

Reply:
Done